apps/src/StudioApp.js

```diff
@@ -1579,8 +1588,8 @@ StudioApp.prototype.handleHideSource_ = function (options) {
       $('body').append(wireframeSendToPhone);
     }
     document.body.style.backgroundColor = '#202B34';
-  // For share page on mobile, do not show this part.
-  } else if (!options.embed && !(this.share && dom.isMobile())) {
+  // For share page, do not show this part.
+  } else if (!options.embed && !this.share) {
```

Comment:
Do you know if we ever still hit this block? i.e. is there a scenario in which we're calling handleHideSource_ but both embed and share are false?